### PR TITLE
Update free-programming-books-langs.md

### PR DESCRIPTION
**Pull request by MrCyGuy**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

Add resources

### Description

I wanted to include more free books on COBOL, so that's just what I did. I added 5 free books about programming in COBOL.

### Why is this valuable (or not)?

I've nearly doubled the amount of resources there are now for COBOL. This expands the amount of learning, which is the purpose of this repository. 

### How do we know it's really free?

I can access these books for free without any information inputs or money paid.

### For book lists, is it a book? For course lists, is it a course? etc.

All my added resources are books. I have only included books.

## Checklist:
- [X] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [X] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [X] Include author(s) and platform where appropriate.
- [X] Put lists in alphabetical order, correct spacing.
- [X] Add needed indications (PDF, access notes, under construction).
- [X] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

File: books/free-programming-books-langs.md [hunk 1]
```diff
@@ -599,10 +599,15 @@ Books on general-purpose programming that don't focus on a specific language are
...
 
+* [A Guide to COBOL Programming](https://www.ibm.com/docs/en/ssw_ibm_i_71/rzase/sc092540.pdf) - David E. Woolbright (PDF)
+* [COBOL: A Self Instructional Manuel](https://babel.hathitrust.org/cgi/pt?id=mdp.39015000482466&view=1up&seq=14&skin=2021) - James A. Saxon 
+* [COBOL Programmer's Guide and Reference](https://www.microfocus.com/documentation/orbix/orbixfm62/pdfdoc/cobol_guide_and_ref.pdf) - Micro Focus (PDF)
 * [COBOL Programming Fundamental](http://yusman.staff.gunadarma.ac.id/Downloads/files/33460/COBOL_Programming_Fundamental.pdf) (PDF)
+* [COBOL Programming Standards](https://www.tonymarston.net/cobol/cobolstandards.html) - A J Marston (HTML)
 * [Enterprise COBOL for z/OS documentation library](http://www-01.ibm.com/support/docview.wss?uid=swg27036733)
 * [GNU COBOL Programmers Guide](https://edoras.sdsu.edu/doc/GNU_Cobol_Programmers_Guide_2.1.pdf) - Gary L. Cutler (PDF)
 * [ILE COBOL Programmer's Guide](https://www.ibm.com/docs/de/ssw_ibm_i_74/pdf/sc092539.pdf) (PDF)
+* [Learn COBOL Programming](https://riptutorial.com/Download/cobol.pdf) - Michael Coughlan (PDF)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:610]: not the author. see attributions for other riptutorial resources

File: books/free-programming-books-langs.md [hunk 2]
```diff
@@ -599,10 +599,15 @@ Books on general-purpose programming that don't focus on a specific language are
 
 ### COBOL
 
+* [A Guide to COBOL Programming](https://www.ibm.com/docs/en/ssw_ibm_i_71/rzase/sc092540.pdf) - David E. Woolbright (PDF)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:602]: not the author

File: books/free-programming-books-langs.md [hunk 3]
```diff
@@ -599,10 +599,15 @@ Books on general-purpose programming that don't focus on a specific language are
...
+* [COBOL: A Self Instructional Manuel](https://babel.hathitrust.org/cgi/pt?id=mdp.39015000482466&view=1up&seq=14&skin=2021) - James A. Saxon
```
Comment (eshellman) [inline books/free-programming-books-langs.md:603]: fix spelling.

File: books/free-programming-books-langs.md [hunk 4]
```diff
@@ -599,10 +599,15 @@ Books on general-purpose programming that don't focus on a specific language are
 
 ### COBOL
 
+* [A Guide to COBOL Programming](https://www.ibm.com/docs/en/ssw_ibm_i_71/rzase/sc092540.pdf) - David E. Woolbright (PDF)
+* [COBOL: A Self Instructional Manuel](https://babel.hathitrust.org/cgi/pt?id=mdp.39015000482466&view=1up&seq=14&skin=2021) - James A. Saxon 
+* [COBOL Programmer's Guide and Reference](https://www.microfocus.com/documentation/orbix/orbixfm62/pdfdoc/cobol_guide_and_ref.pdf) - Micro Focus (PDF)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:604]: that's not the author

**Review by eshellman — CHANGES_REQUESTED** [review]
good additions, but get the authors right!

File: books/free-programming-books-langs.md [hunk 5]
```diff
@@ -599,12 +599,17 @@ Books on general-purpose programming that don't focus on a specific language are
 
 ### COBOL
 
+* [COBOL: A Self Instructional Manual](https://babel.hathitrust.org/cgi/pt?id=mdp.39015000482466&view=1up&seq=14&skin=2021) - James A. Saxon 
+* [Cobol: From Micro To Mainframe](https://vdoc.pub/download/cobol-from-micro-to-mainframe-3rd-edition-5t4osmo5p540) - Robert T. Grauer, Carol Vazquez Villar, Arthur R. Buss (PDF)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:603]: This appears not to be an authorized distribution. Please see docs/CONTRIBUTING.md guidelines.

File: books/free-programming-books-langs.md [hunk 6]
```diff
@@ -599,12 +599,17 @@ Books on general-purpose programming that don't focus on a specific language are
...
 * [COBOL Programming Fundamental](http://yusman.staff.gunadarma.ac.id/Downloads/files/33460/COBOL_Programming_Fundamental.pdf) (PDF)
+* [COBOL Programming Standards](https://www.tonymarston.net/cobol/cobolstandards.html) - A J Marston (HTML)
 * [Enterprise COBOL for z/OS documentation library](http://www-01.ibm.com/support/docview.wss?uid=swg27036733)
 * [GNU COBOL Programmers Guide](https://edoras.sdsu.edu/doc/GNU_Cobol_Programmers_Guide_2.1.pdf) - Gary L. Cutler (PDF)
 * [ILE COBOL Programmer's Guide](https://www.ibm.com/docs/de/ssw_ibm_i_74/pdf/sc092539.pdf) (PDF)
 * [Micro Focus: OO Programming with Object COBOL for UNIX (1999)](https://www.microfocus.com/documentation/object-cobol/oc41books/oppubb.htm) - MERANT International Ltd. (HTML)
 * [OpenCOBOL 1.1 - Programmer's Guide](http://open-cobol.sourceforge.net/guides/OpenCOBOL%20Programmers%20Guide.pdf) (PDF)
+* [Sams Teach Yourself COBOL in 21 Days](http://testa.roberta.free.fr/My%20Books/Mainframe/Cobol/Teach%20Yourself%20Cobol%20In%2021%20Days%20(2nd%20Ed).pdf) - Mo Budlong (PDF)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:611]: This appears not to be an authorized distribution. Please see docs/CONTRIBUTING.md guidelines.

File: books/free-programming-books-langs.md [hunk 7]
```diff
@@ -599,12 +599,17 @@ Books on general-purpose programming that don't focus on a specific language are
 
 ### COBOL
 
+* [COBOL: A Self Instructional Manual](https://babel.hathitrust.org/cgi/pt?id=mdp.39015000482466&view=1up&seq=14&skin=2021) - James A. Saxon
```
Comment (eshellman) [inline books/free-programming-books-langs.md:602]:
```suggestion
* [COBOL: A Self Instructional Manual](https://babel.hathitrust.org/cgi/pt?id=mdp.39015000482466&view=1up&seq=9&skin=2021) - James A. Saxon 
```

**Review by eshellman — CHANGES_REQUESTED** [review]
unauthorized distributions tend to disappear, so we don't allow them

**Comment by MrCyGuy** [comment]
So I removed the unauthorized distribution books and instead replaced them with additions in Python, Pascal, and CSS. I hope this works! Please let me know if it doesn't and thank you for helping me get these right!

File: books/free-programming-books-langs.md [hunk 8]
```diff
@@ -1975,6 +1979,7 @@ Books on general-purpose programming that don't focus on a specific language are
 * [Code Like a Pythonista: Idiomatic Python](https://web.archive.org/web/20180411011411/http://python.net/~goodger/projects/pycon/2007/idiomatic/handout.html) - David Goodger *(:card_file_box: archived)*
 * [Composing Programs](http://composingprograms.com) (3.x)
 * [Computational and Inferential Thinking](https://inferentialthinking.com/chapters/intro.html) - Ani Adhikari, John DeNero, David Wagner (HTML)
+* [Computational Physics with Python](https://belglas.com/wp-content/uploads/2018/03/cpwp.pdf) - Eric Ayars (PDF)
```
Comment (eshellman) [inline books/free-programming-books-langs.md:1982]: There's no indications that this is an authorized distribution. Please see docs/CONTRIBUTING.md guidelines.

**Review by eshellman — CHANGES_REQUESTED** [review]
remove the python book and you're good to go

**Comment by MrCyGuy** [comment]
Hey can you review if my PR is good to go?

**Comment by eshellman** [comment]
Thanks, and welcome to free-programming-books!

**Pull request merged**